src/nativeTrackerManager.js

```diff
@@ -103,7 +122,30 @@ export function newNativeTrackerManager(win) {
     }
   }
 
+  //Native Render Ad Function
+  let renderNativeAd = function(nativeTag){
+    window.pbNativeData = nativeTag;
+    const nativeAssetManager = newNativeAssetManager(window);
+
+    if(nativeTag.hasOwnProperty('adId')) {
+      if(nativeTag.hasOwnProperty('rendererUrl') && !nativeTag.rendererUrl.match(/##.*##/i)){
+        const scr = document.createElement('SCRIPT');
+        scr.src = nativeTag.rendererUrl,
+        scr.id = 'pb-native-renderer';
+        document.body.appendChild(scr);
+      }
+      nativeAssetManager.loadAssets(nativeTag.adId,fireNativeCallback);
+      fireNativeCallback();
+      fireNativeImpTracker(nativeTag.adId);
+    }else{
+      console.warn('Prebid Native Tag object was missing \'adId\'.');
+    }
+
+    
+  }
+
   return {
-    startTrackers
+    startTrackers,
+    renderNativeAd
```

Comment:
Suggest move `renderNativeAd` to a new class instead of add into `newNativeTrackerManager`. In long run, `newNativeTrackerManager.startTrackers` will get deprecated. If we separate code `renderNativeAd` from `newNativeTrackerManager` now, it will be much easier when that happens.

Reply:
renderAd code separated into new nativeRenderManager